candidate_models/model_commitments/__init__.py

```diff
@@ -1,9 +1,14 @@
+from candidate_models.base_models import base_model_pool
+
 from candidate_models.model_commitments.cornets import cornet_brain_pool
-from candidate_models.model_commitments.ml_pool import ml_brain_pool
-from candidate_models.utils import UniqueKeyDict
+from candidate_models.model_commitments.model_layer_def import model_layers
+from submission.ml_pool import MLBrainPool
+from submission.utils import UniqueKeyDict
 
 brain_translated_pool = UniqueKeyDict()
 
+ml_brain_pool = MLBrainPool(base_model_pool, model_layers)
```

Comment:
doesn't this replace the `brain_translated_pool`, i.e. `brain_translated_pool = MLBrainPool(base_model_pool, model_layers)` and remove `ml_brain_pool = `?

Also, the pool here is really only needed for unit tests right? Maybe we should move it there then

Reply:
I agree, I changed it

Reply:
doesn't appear to be changed on my end, there's still both variables (`ml_brain_pool` and `brain_translated_pool`)

Reply:
I would not delete the `brain_translated_pool` as it is kind of the combination of all available models. Very convenient when people want to run models locally. It is true that I can get rid of this `ml_brain_pool` variable, which I did. But then you commented you'd rather have it expanded so I reverted the change :woman_shrugging:  The comments get a bit out of hand.

Reply:
ah got it, thanks for explaining